src/types/lib/ConsiderationConstants.sol

```diff
@@ -300,6 +300,40 @@ uint256 constant OrderFulfilled_offer_body_offset = 0x80;
 uint256 constant OrderFulfilled_consideration_head_offset = 0x60;
 uint256 constant OrderFulfilled_consideration_body_offset = 0x120;
 
+/*
+ * 3 memory slots/words for `authorizeOrder` and `validateOrder` calldata
```

Comment:
note: 3 because extra data is empty, and order hashes has a single element

Reply:
Exactly. Perhaps we can add to the comments to make it more clear.

cc: @0age

Reply:
I did this in the other PR